qt5ct.tera

```diff
@@ -0,0 +1,18 @@
+---
+whiskers:
+  version: "2.5.0"
+  matrix:
+    - flavor
+    - accent
+  filename: "themes/qt5ct/Catppuccin-{{ flavor.identifier | capitalize }}-{{accent | capitalize }}.conf"
```

Comment:
```suggestion
  filename: "themes/Catppuccin-{{ flavor.identifier | capitalize }}-{{accent | capitalize }}.conf"
```

Reply:
so qt6ct themes should be a separate repo?

Reply:
qt5ct and qt6ct theme is the same except for

- qt5ct - 21x3 color fields
- qt6ct - 22x3 color fields

Reply:
Is that like the next version? I imagine we would leave behind a branch with just the v5 themes and then update main to v6. Though, this repository is named specifically with for v5 so...

Reply:
That would be a more meta issue for @sgoudham. I'm not sure if we would rename the repository to something more general like `qtct`, or just note that we support both in the readme, or make a new repository entirely...

Reply:
yup, ok 
for now i'll just remove the folder

Reply:
just opened this on the main catppuccin github discussion page: [https://github.com/catppuccin/catppuccin/discussions/2624](https://github.com/catppuccin/catppuccin/discussions/2624)